api-report/garbage-collector.api.md

```diff
@@ -5,11 +5,16 @@
 ```ts
 
 import { IGarbageCollectionData } from '@fluidframework/runtime-definitions';
+import { IGarbageCollectionState } from '@fluidframework/runtime-definitions';
+import { IGarbageCollectionSummaryDetails } from '@fluidframework/runtime-definitions';
 import { ITelemetryLogger } from '@fluidframework/common-definitions';
 
 // @public
 export function cloneGCData(gcData: IGarbageCollectionData): IGarbageCollectionData;
 
+// @public
+export function concatGarbageCollectionState(destGCState: IGarbageCollectionState, srcGCState: IGarbageCollectionState): void;
```

Comment:
should this be a add() method on IGarbageCollectionState ?

Reply:
Or maybe return the result of addition (functional)?

Reply:
Updated it to return the result.